resources/assets/lib/beatmap-discussions/editor.tsx

```diff
@@ -394,8 +414,17 @@ export default class Editor extends React.Component<Props, State> {
       }
     }
 
-    Transforms.deselect(this.slateEditor);
-    this.onChange(this.emptyDocTemplate);
+    this.resetEditorValue();
+  };
+
+  // Overrides slate Editable's defaultScrollSelectionIntoView and always use browser's version
+  // There's something wrong with slate's range detection in the prior calling function that causes
+  // letters and numbers to be differently.
+  scrollSelectionIntoView = (editor: ReactEditor, domRange: DOMRange) => {
+    domRange.startContainer.parentElement?.scrollIntoView({
```

Comment:
how does this work in safari?

Reply:
Safari supports `scrollIntoView(ScrollOptions)` since 15.4
https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes#Web-API